extensions/funqy/funqy-knative-events/runtime/src/main/java/io/quarkus/funqy/knative/events/CloudEventOutput.java

```diff
@@ -0,0 +1,31 @@
+package io.quarkus.funqy.knative.events;
+
+import java.lang.reflect.Type;
+
+public class CloudEventOutput<T> {
+    Type javaType;
+    String type;
+    String source;
+    T data;
+
+    protected CloudEventOutput() {
+
+    }
+
+    public Type javaType() {
```

Comment:
Is this used for anything?

Reply:
See the new issue linked in ther comment